### PR TITLE
[FEATURE] Cache invalidation for address records

### PR DESCRIPTION
**Pull request by christophlehmann**
This sets page cache tags for address records

### CODE REVIEW

File: Classes/Utility/CacheUtility.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+
```
Comment (georgringer) [inline Classes/Utility/CacheUtility.php]: please add strict type for new classes

File: Tests/Unit/Utility/CacheUtilityTest.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?php
+
```
Comment (georgringer) [inline Tests/Unit/Utility/CacheUtilityTest.php]: please add strict type for new classes

File: Classes/Utility/CacheUtility.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+/**
+ * This file is part of the "tt_address" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+use FriendsOfTYPO3\TtAddress\Domain\Model\Address;
+
+/**
+ * Class CacheUtility
```
Comment (georgringer) [inline Classes/Utility/CacheUtility.php]: would be nice to have a oneliner like Handling cache tags

File: Classes/Utility/CacheUtility.php (hunk 1 above)
Comment (christophlehmann) [reply Classes/Utility/CacheUtility.php]: Good catch, that makes sense

**Pull request merged**